src/allotropy/parsers/chemometec_nc_view/chemometec_nc_view_reader.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+
+from allotropy.named_file_contents import NamedFileContents
+from allotropy.parsers.utils.pandas import (
+    df_to_series_data,
+    read_csv,
+    SeriesData,
+)
+
+
+class ChemometecNcViewReader:
+    SUPPORTED_EXTENSIONS = "xlsx,csv"
+    header: SeriesData
+    data: pd.DataFrame
+
+    def __init__(self, named_file_contents: NamedFileContents) -> None:
+        self.data = read_csv(named_file_contents.contents)
```

Comment:
What if the data is xlsx? I think I remember this discussion from standup - if we don't have an xlsx example, you can just remove it from supported extensions for now.